UI/static/css/main.css

```diff
@@ -0,0 +1,71 @@
+html, body{
+    height: 100%;
+    width: 100%;
+    overflow: auto;
+}
+
+body{
+    position: relative;
+    margin: 0;
+}
+
+.logo{
+    background: cyan;
+    border-radius: 50%;
+    height: 100px;
+    line-height: 100px;
+    text-align: center;
+    vertical-align: middle;
+    width: 100px;
+    display: inline-block;
+
```

Comment:
Make sure to format your css properly, don't leave blank lines everywhere this makes the file look untidy and tiring to read. Also mind spacing, there should be a space between a class name and the opening curly brace.